assets/queries/k8s/deployment_without_pod_disruption_budget/query.rego

```diff
@@ -3,26 +3,28 @@ package Cx
 CxPolicy[result] {
 	deployment := input.document[i]
 	deployment.kind == "Deployment"
+	deployment.spec.replicas > 1
 	metadata := deployment.metadata
 
-	not CheckIFPdbExists(deployment)
+	CheckIFPdbExists(deployment) == false
 
 	result := {
 		"documentId": input.document[i].id,
-		"searchKey": sprintf("metadata.name=%s", [metadata.name]),
+		"searchKey": sprintf("metadata.name={{%s}}.spec.selector.matchLabels", [metadata.name]),
 		"issueType": "MissingAttribute",
-		"keyExpectedValue": sprintf("metadata.name=%s are targeted by a PDB", [metadata.name]),
-		"keyActualValue": sprintf("metadata.name=%s are not targeted by a PDB", [metadata.name]),
+		"keyExpectedValue": sprintf("metadata.name=%s is targeted by a PodDisruptionBudget", [metadata.name]),
+		"keyActualValue": sprintf("metadata.name=%s is not targeted by a PodDisruptionBudget", [metadata.name]),
 	}
 }
 
 CheckIFPdbExists(deployments) = result {
 	documents := input.document
 	pdbs := [pdb | documents[index].kind == "PodDisruptionBudget"; pdb = documents[index]]
```

Comment:
Don't iterate through every document, the query should look for PodDisruptionBudget inside the same document to prevent timeouts